build/ProjectReunion-CI.yml

```diff
@@ -50,8 +55,9 @@ jobs:
 - template: AzurePipelinesTemplates\ProjectReunion-CreateNugetPackage-Job.yml
   parameters:
     jobName: CreateNugetPackage
-    dependsOn: Build
+    dependsOn: PublishMRT
```

Comment:
>dependsOn [](start = 4, length = 9)

the dependency in release pipeline also needs change, but that can be in another PR.